pkgs/os-specific/linux/efa/cmake-fixups.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f30f7c4..471d6c8 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
```

Comment:
@mrgolin mind if I change this upstream? there is no way currently to provide `-DKERNEL_DIR=/path/to/modules/version/build` to the top-level CMakeLists.txt, it is hardcoded to always be rooted under /lib. 

But under NixOS that directory is expected to be under the nix store, (ie: `/nix/store/v4mj9kdrml07y8g1lpc02k0yxwf69y93-linux-6.12.1-dev/lib/modules/6.12.1/build/`) so this is necessary.